src/Program.ts

```diff
@@ -1472,3 +1553,10 @@ export interface FileTranspileResult {
     map: SourceMapGenerator;
     typedef: string;
 }
+
+export function test(foo) {
+    const bar = foo + 'Hello';
+
+
+    return bar;
+}
```

Comment:
This shouldn't be here I feel

Reply:
oops! Good catch. 

https://github.com/rokucommunity/brighterscript/pull/1432